docs/reference/transfers.md

```diff
@@ -42,100 +41,38 @@ TigerBeetle uses the same data structures internally and
 externally. This means that sometimes you need to set temporary values
 for fields that TigerBeetle, not you (the user), are responsible.
 
-### Single-Phase Transfer
+### Single-phase transfer
 
 Single-phase transfers post funds to accounts immediately when they are created.
 
-### Two-Phase Transfer
+### Two-phase transfer
 
 A pending transfer followed by a post-pending transfer, void-pending transfer, or a timeout is
 called a "two-phase transfer". Unlike a single-phase transfer, a two-phase transfer moves funds in
 stages:
 
-1. [Reserve funds](#pending-transfer).
-2. Resolve funds (post, void, or timeout).
-
-Pending amounts (the pending transfer's [`amount`](#amount)) cannot be spent by either the payer or
-payee until the pending transfer is _resolved_ — that is, until the first of the following events
-occur:
-
-* If a corresponding [`post_pending_transfer`](#post-pending-transfer) is
-  committed, some or all of the pending transfer's reserved funds are
-  posted, and the remainder (if any) is restored to the original
-  accounts.
-* If a corresponding [`void_pending_transfer`](#void-pending-transfer) is
-  committed, the pending transfer's reserved funds are restored to
-  their original accounts.
-* If the pending transfer has a timeout and the timeout expires, the pending
-  transfer's reserved funds are restored to their original accounts.
+1. Reserve funds
+2. Resolve funds (post, void, or timeout)
 
-A pending transfer resolves at most once.
-Attempting to resolve a pending transfer more than once will return the applicable error result:
-- [`pending_transfer_already_posted`](./operations/create_transfers.md#pending_transfer_already_posted)
-- [`pending_transfer_already_voided`](./operations/create_transfers.md#pending_transfer_already_voided)
-- [`pending_transfer_expired`](./operations/create_transfers.md#pending_transfer_expired)
```

Comment:
I think this is work keeping in the reference doc.

Reply:
Added back